src/static/css/content.less

```diff
@@ -279,6 +276,10 @@ figure {
 
```

Comment:
@designlanguage No, which is actually a more serious responsive issue than the one I screenshotted, but I agree that that can be remedied more easily with CF4 in place.